code/lib/cli/src/initiate.ts

```diff
@@ -320,16 +323,10 @@ async function doInitiate(options: CommandOptions, pkg: PackageJson): Promise<vo
 
     // Add a new line for the clear visibility.
     logger.log();
-    return;
+    throw new HandledError(`Angular project already installed`);
   }
 
-  const installResult = await installStorybook(
-    projectType as ProjectType,
-    packageManager,
-    options
-  ).catch((e) => {
-    process.exit();
-  });
```

Comment:
I'm not sure why this was here, given the outer catch does the same thing with `exit(1)`. Did we want this to exit 0? I doubt it.